src/components/ReportActionItem/ReportPreview.js

```diff
@@ -113,6 +117,7 @@ function ReportPreview(props) {
     const iouCanceled = ReportUtils.isArchivedRoom(props.chatReport);
     const numberOfRequests = ReportActionUtils.getNumberOfMoneyRequests(props.action);
     const moneyRequestComment = lodashGet(props.action, 'childLastMoneyRequestComment', '');
+    const isReportDraft = ReportUtils.isPolicyExpenseChat(props.chatReport) && ReportUtils.isReportDraft(props.iouReport);
```

Comment:
The isPolicyExpenseChat is also used [here](https://github.com/Expensify/App/blob/c5e1049208b61aa319d2b4a109d02bae6546eabe/src/components/ReportActionItem/ReportPreview.js#L160) could we dry this up?